.github/workflows/ci.yml

```diff
@@ -10,6 +10,8 @@ on:
 
 env:
   NXF_ANSI_LOG: false
+  CAPSULE_LOG: none
+  NFTEST_VER: "0.8.4"
```

Comment:
not using sharding or latest nf-test due to the issue with download limit from mirBase